apps/parsley/src/constants/logURLTemplates.ts

```diff
@@ -127,8 +127,16 @@ const getEvergreenTaskFileURL = (
   fileName: string,
 ) => `${evergreenURL}/task_file_raw/${taskID}/${execution}/${fileName}`;
 
+const getCompleteLogsURL = (
```

Comment:
To match the other function names in this file; this function also needs a JSDoc comment
```suggestion
const getEvergreenCompleteLogsURL = (
```